builds.go

```diff
@@ -0,0 +1,161 @@
+package gogitlab
+
+import (
+	"encoding/json"
+	"io"
+)
+
+const (
+	project_builds          = "/projects/:id/builds"                         // List project builds
+	project_build           = "/projects/:id/builds/:build_id"               // Get a single build
+	project_commit_builds   = "/projects/:id/repository/commits/:sha/builds" // List commit builds
+	project_build_artifacts = "/projects/:id/builds/:build_id/artifacts"     // Get build artifacts
+	project_build_cancel    = "/projects/:id/builds/:build_id/cancel"        // Cancel a build
+	project_build_retry     = "/projects/:id/builds/:build_id/retry"         // Retry a build
+	project_build_erase     = "/projects/:id/builds/:build_id/erase"         // Erase a build
+)
+
+type ArtifactsFile struct {
+	Filename string `json:"filename"`
+	Size     int    `json:"size"`
+}
+
+type Build struct {
+	ID            int           `json:"id"`
+	ArtifactsFile ArtifactsFile `json:"artifacts_file"`
+	Commit        Commit        `json:"commit"`
+	Coverage      interface{}   `json:"coverage"`
+	CreatedAt     string        `json:"created_at"`
+	DownloadURL   string        `json:"download_url"`
+	FinishedAt    string        `json:"finished_at"`
+	Name          string        `json:"name"`
+	Ref           string        `json:"ref"`
+	Runner        interface{}   `json:"runner"`
+	Stage         string        `json:"stage"`
+	StartedAt     string        `json:"started_at"`
+	Status        string        `json:"status"`
+	Tag           bool          `json:"tag"`
+	User          User          `json:"user"`
+}
```

Comment:
`runner` and `coverage` are really `interface{}` ? i only see `null` in tests. what types can they be in real world :smile: ?